ui/ui-frontend/projects/pastis/src/app/profile/profile-preview/profile-information-tab/profile-information-tab/profile-information-tab.component.ts

```diff
@@ -98,7 +94,28 @@ export class ProfileInformationTabComponent {
     }
   }
 
+  canSubmit() {
+    return this.form.valid && !this.submited && this.formHasChanged()
+  }
+
+  formHasChanged() {
+    // tslint:disable-next-line:forin
```

Comment:
attention : à ne pas désactiver la règle tslint

Reply:
la regle c'est for in il faut un if en premiere ligne   
je pense qu'on peut la supprimer tout court de tslint

Reply:
parceque j'ai un if mais il est en 2em ligne  👍